plugins/pulp_rpm/plugins/importers/yum/existing.py

```diff
@@ -129,6 +129,20 @@ def check_all_and_associate(wanted, conduit, download_deferred):
     return still_wanted
 
 
+def update_unit_downloaded_flag(unit):
+    """
+    Updates unit's download flag to True once it was placed to its storage location.
+
+    :param unit: A content unit.
+    :type unit: pulp_rpm.plugins.db.models.RPM/DRPM/Distribution
+    """
+
+    unit_model = plugin_api.get_unit_model_by_id(unit._content_type_id)
```

Comment:
Class attributes are also present on instances, so you can just do ``unit.objects`` without having to get the model class from the API. It feels unintuitive for some reason; I also did something like you did. In my case, I think I accessed something like ``unit.__class__.objects``, and someone pointed out that I could access ``objects`` directly.